libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -91,13 +87,15 @@ td {
 }
 
 .day {
-  $dayWidth: 40px;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  $dayWidth: size('xl');
```

Comment:
Very minor thing, but could you move declaration of Sass variables to the top of the scope?